src/functionalTest/resources/test-data/new-application-payments/metadata.json

```diff
@@ -1,5 +1,4 @@
 {
-  "case_number": "1111222233334446",
```

Comment:
instead of completely removing can we do `"case_number":""` as that is what we get from our supplier